BALSAMIC/containers/msisensorpro/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM ubuntu:22.04
+
+LABEL base.image="ubuntu:22.04" \
+      maintainer="Clinical Genomics" \
+      about.contact="support@clinicalgenomics.se" \
+      software="msisensor-pro" \
+      software.version="1.2.0" \
+      about.summary="Microsatellite Instability (MSI) detection using high-throughput sequencing data" \
+      about.home="https://github.com/xjtu-omics/msisensor-pro" \
+      about.documentation="https://github.com/xjtu-omics/msisensor-pro/wiki" \
+      about.license="MIT License (MIT)"
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get -y upgrade && \
+    apt-get -y install -y --no-install-recommends git make build-essential \
```

I think we have an extra `-y` flag here:

```suggestion
    apt-get -y install --no-install-recommends git make build-essential \
```

---

I am sorry. It was missed. Thanks for reminding me.